src/sidebar/components/Annotation/AnnotationLicense.tsx

```diff
@@ -7,13 +11,14 @@ export default function AnnotationLicense() {
   return (
     <div className="pt-2 border-t text-sm leading-none">
       <Link
-        classes="flex items-center text-color-text-light"
+        classes="flex items-center"
+        color="text-light"
         href="http://creativecommons.org/publicdomain/zero/1.0/"
         target="_blank"
         title="View more information about the Creative Commons Public Domain dedication"
       >
-        <Icon name="cc-std" classes="text-tiny" />
-        <Icon name="cc-zero" classes="ml-px text-tiny" />
+        <CcStdIcon className="w-em h-em text-tiny" />
+        <CcZeroIcon className="w-em h-em ml-px text-tiny" />
         <div className="ml-1">
           Annotations can be freely reused by anyone for any purpose.
```

Comment:
This text currently has slightly weak link-ness cues as there is only a hover color and no underline. This was true before the conversion though.

Reply:
Yeah I know...